app.js

```diff
@@ -1,9 +1,12 @@
 const express = require("express")
 const app = express()
-const { getTopics } = require("./controllers/news_controller")
+const { 
+    getTopics,
+    getArticles
+} = require("./controllers/news_controller")
```

Comment:
Please separate your controllers and models out into the different data that they're utilising e.g. topicsController, commentsContoller